lib/absinthe/blueprint/schema.ex

```diff
@@ -169,7 +169,6 @@ defmodule Absinthe.Blueprint.Schema do
   end
 
   defp build_types([:close | rest], [%Schema.InputValueDefinition{} = arg, field | stack], buff) do
-    arg = Map.update!(arg, :default_value, fn val -> {:unquote, [], [val]} end)
```

Comment:
Seemed to not work, and i'm guess it was supposed to do the evaluation. Clearly it didn't